### PR TITLE
Ignition 8 dev

### PR DESCRIPTION
**Pull request by TomKimsey**

### CODE REVIEW

File: eWonConnector-gateway/src/main/java/org/imdc/ewon/SyncManager.java [hunk 1]
```diff
@@ -244,14 +244,14 @@ protected void updateLive() {
                 }
             }
         } catch (InterruptedException e) {
-            logger.error("Error while reading tag parameters. InterruptedException.");
+            logger.error("Error while reading tag parameters. InterruptedException.", e);
```
Comment (alexjhawk) [inline eWonConnector-gateway/src/main/java/org/imdc/ewon/SyncManager.java]: Are these whitespace changes?

Comment (alexjhawk) [reply eWonConnector-gateway/src/main/java/org/imdc/ewon/SyncManager.java]: nevermind

File: eWonConnector-gateway/src/main/java/org/imdc/ewon/SyncManager.java [hunk 2]
```diff
@@ -267,9 +267,8 @@ protected void updateLive() {
                     try {
                         // Value is a string
                         if (valueString.charAt(0) == '\"') {
-                            // Strip the "s from the string
-                            // TODO: Consider the case where the string value has "s
-                            value = valueString.replaceAll("\"", "");
+                            // Strip leading and trailing double quote chars
```
Comment (alexjhawk) [inline eWonConnector-gateway/src/main/java/org/imdc/ewon/SyncManager.java]: Is there any case where the first character may be a quote, but there is not a quote as the last character?

Comment (TomKimsey) [reply eWonConnector-gateway/src/main/java/org/imdc/ewon/SyncManager.java]: no, quotes can only occur in strings which by definition in the m2web api start and end with "

Comment (alexjhawk) [reply eWonConnector-gateway/src/main/java/org/imdc/ewon/SyncManager.java]: ok, perfect

**Pull request merged**